teku/src/main/java/tech/pegasys/teku/cli/options/BeaconRestApiOptions.java

```diff
@@ -50,10 +50,9 @@
   private String restApiInterface = "127.0.0.1";
 
   @Option(
-      names = {"--rest-api-host-whitelist"},
+      names = {"--rest-api-host-allowlist", "--rest-api-host-whitelist"},
       paramLabel = "<hostname>",
-      description =
-          "Comma separated list of hostnames to whitelist for access, or * to accept any host",
+      description = "Comma separated list of hostnames to allow, or * to allow any host",
       split = ",",
       arity = "0..*")
   private final List<String> restApiHostWhitelist = Arrays.asList("127.0.0.1", "localhost");
```

Comment:
we can rename this variable correctly now so that we don't have to rename it later.

Reply:
done